README.md

```diff
@@ -82,14 +82,28 @@ the ``--count`` option. For example, to run a test class 10 times:
 
     pytest --count=10 --cassandra-dir=~/path/to/cassandra pending_range_test.py
 
+Use ``--log-cli-level=DEBUG`` to get log messages emitted from the test
+
+    pytest --log-cli-level=DEBUG --cassandra-dir=~/path/to/cassandra pending_range_test.py
+
 Existing tests are probably the best place to start to look at how to write
 tests.
 
+Node reusage 4.0+
+-----------------
 Each test spawns a new fresh cluster and tears it down after the test. If a
 test fails, the logs for the node are saved in a `logs/<timestamp>` directory
 for analysis (it's not perfect but has been good enough so far, I'm open to
 better suggestions).
 
+Starting at 4.0, as dtests can be very heavy, in order to speed up things node reusage has been implemented. Basically a 
```

Comment:
Recommend rephrasing this - the above paragraph states "Each test spawns a new fresh cluster..." and then this paragraph refutes it. Consider something like "In the default configuration, each test spawns a new fresh cluster", then revising this paragraph to read "Optionally starting at 4.0...".

We'd also need to make this patch optional to make that true as well, but I left another comment about how I think that's the right path.